x-pack/plugins/infra/server/lib/log_analysis/log_entry_rate_analysis.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { pipe } from 'fp-ts/lib/pipeable';
+import { map, fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+import { getJobId } from '../../../common/log_analysis';
+import { throwErrors, createPlainError } from '../../../common/runtime_types';
+import { KibanaFramework } from '../adapters/framework/kibana_framework_adapter';
+import { NoLogAnalysisResultsIndexError } from './errors';
+import {
+  logRateModelPlotResponseRT,
+  createLogEntryRateQuery,
+  LogRateModelPlotBucket,
+  CompositeTimestampPartitionKey,
+} from './queries';
+import { RequestHandlerContext, KibanaRequest } from '../../../../../../src/core/server';
```

Comment:
This was corrected as

```suggestion
import { RequestHandlerContext, KibanaRequest } from 'src/core/server';
```

in other places.